croptalk/model_openai_functions.py

```diff
@@ -135,7 +136,20 @@ def _get_doc_retriever_tool(self) -> StructuredTool:
         """
         find_docs = StructuredTool.from_function(
             name="FindDocs",
-            description="Searches and returns information given the filters.",
+            description="This tool is used to find information contained in the Crop Insurance Handbook (CIH),"
+                        "in the Basic Provisions documents (BP) and within the Crop Provision documents. It should not"
```

Comment:
Should we add `(CP)` abbreviation for Crop Provision documents liek we did for CIH and BP?